Rocket.Chat/Views/Cells/Subscription/SubscriptionCellCondensed.swift

```diff
@@ -0,0 +1,213 @@
+//
+//  SubscriptionCellCondensed.swift
+//  Rocket.Chat
+//
```

Comment:
@rafaelks This class shares a lot of code with the `SubscriptionsCell`, have you considered subclassing?

Reply:
@Sameesunkaria Yes... we could have used subclass here maybe.

Reply:
@rafaelks How about we refactor it later?

Reply:
@Sameesunkaria Yes, let's do it later in a refactor. 👍